src/form-context.tsx

```diff
@@ -151,15 +163,22 @@ export const useFieldProps = ({
         value,
         defaultValue: defaultValue ?? (value === undefined ? rawValues[name] ?? '' : undefined),
         name,
+        label,
         helperText: formErrors[name] || helperText,
         required: !optional,
         error: error || !!formErrors[name],
         disabled: disabled || formStatus === 'sending',
         onBlur: (e: React.FocusEvent) => {
-            setFormError({name, error: validate?.(values[name], rawValues[name])});
+            let error: string | undefined;
+            if (!rawValues[name] && !optional) {
+                error = texts.formFieldErrorIsMandatory || t(tokens.formFieldErrorIsMandatory);
+            } else if (validate) {
+                error = validate(values[name], rawValues[name]);
+            }
+            setFormError({name, error});
             onBlur?.(e);
         },
-        inputRef: (input: HTMLInputElement | null) => register(name, {input, validator: validate}),
+        inputRef: (input: HTMLInputElement | null) => register(name, {input, validator: validate, label}),
```

Comment:
I need the field label to be in the registrations map, to be able to use it when rendering the global form error: "Revisa los errores en los siguientes campos: nombre, email"